cdap-master/src/main/java/co/cask/cdap/data/runtime/main/TransactionServiceTwillRunnable.java

```diff
@@ -93,6 +97,7 @@ protected Injector doInit(TwillContext context) {
   @Override
   public void addServices(List<? super Service> services) {
     services.add(injector.getInstance(TransactionService.class));
+    services.add(injector.getInstance(TransactionHttpService.class));
```

Comment:
Added a commit so that TransactionHttpService returns the status of the transaction service, instead of just the handler being discoverable.